deltacat/examples/hello_world.py

```diff
@@ -0,0 +1,27 @@
+import ray
+import deltacat
+import daft
+import pyiceberg
+
+
+def print_package_version_info():
+    print(f"DeltaCAT Version: {deltacat.__version__}")
+    print(f"PyIceberg Version: {pyiceberg.__version__}")
+    print(f"Ray Version: {ray.__version__}")
+    print(f"Daft Version: {daft.__version__}")
+
+
+@ray.remote
+def hello_worker():
+    print("Hello, Worker!")
+    print_package_version_info()
+
+
+def run():
+    print("Hello, Driver!")
+    print_package_version_info()
+    hello_worker.remote()
+
+
+if __name__ == "__main__":
+    run()
```

Comment:
I just frequently rely on this myself as a sanity check to make sure everything in my environment is installed and setup correctly (since a lot of weird errors in remote environments have historically just come from running different versions of Ray/Daft/DeltaCAT/PyIceberg/etc. remotely than you were running locally).

I think this and the logging example probably point to a need of separating out `dev_examples` useful more for DeltaCAT devs from customer-facing `examples`.